doc/nrf/gs_ins_mac.rst

```diff
@@ -64,3 +64,38 @@ Install Homebrew and install the required tools using the ``brew`` command line
 .. include:: gs_ins_windows.rst
    :start-after: buildenv_start
    :end-before: buildenv_end
+
+.. _additional_mac_setup:
+
+Setting up executables on macOS
+===============================
+
+If you start SES on macOS by running the file :file:`bin/emStudio`, make sure to complete the following steps:
+
+1. Specify the path to all executables under :guilabel:`Tools` -> :guilabel:`Options` (in the :guilabel:`nRF Connect` tab).
+
+    .. figure:: images/ses_options.png
+         :alt: nRF Connect SDK options in SES on Windows
+
+         nRF Connect SDK options in SES (Windows)
+
+   Use this section to change the SES environment settings later as well.
+
+#. Specify the path to the west tool as additional CMake option, replacing *path_to_west* with the path to the west executable (for example, ``/usr/local/bin/west``):
+
+    .. parsed-literal::
+       :class: highlight
+
+       -DWEST=\ *path_to_west*
+
+
+You can also start SES on macOS from the command line environment.
+Unlike on Windows and Linux, this is the only case when macOS uses the global PATH variable to find executables.
+If you get an error that a tool or command cannot be found, first make sure that the tool is installed.
```

Comment:
I suggest to have this as a separate paragraph and update it to cover both ways of running SES (using the SES configuration works in both cases).
This sentence can stay as is, the next should be something like: 

If it is installed, verify that its path is configured correctly in the SES settings or in the PATH variable.